js/components/time.js

```diff
@@ -23,6 +23,8 @@ Fliplet.FormBuilder.field('time', {
   },
   data: function() {
     return {
+      timepicker: null,
+      LOCAL_FORMAT: moment.localeData().longDateFormat('LT'),
```

Comment:
I already reviewed this in a different PR. Please open this PR without these changes once the other is merged.